docs/reference/query-dsl/intervals-query.asciidoc

```diff
@@ -101,6 +101,24 @@ If specified, then match intervals from this field rather than the top-level fie
 The `prefix` will be normalized using the search analyzer from this field, unless
 `analyzer` is specified separately.
 
+[[intervals-wildcard]]
+==== `wildcard`
+
+The `wildcard` rule finds terms that match a wildcard pattern.  The pattern will
+expand to match at most 128 terms; if there are more matching terms in the index,
+then an error will be returned.
+
+[horizontal]
+`pattern`::
+Match terms starting matching this pattern
```

Comment:
Can you add the operators that are supported (`?` and `*` ?) and maybe a warning regarding leading wildcards ?